tests/e2e/testcases/subnet/sov/etna/suite.go

```diff
@@ -25,29 +25,32 @@ const (
 	testLocalNodeName = "e2eSubnetTest-local-node"
 )
 
-func createEtnaSubnetEvmConfig() {
+func createEtnaSubnetEvmConfig(poa, pos bool) {
```

Comment:
this one I am not sure about.
Do we need 2 boolean vars? 

I think we assume that if not PoA, it's PoS 
so only 1 boolean var should be sufficient imho

pls feel free to ignore this comment :)